test/invoice_test.rb

```diff
@@ -7,7 +7,7 @@ class InvoiceTest < MiniTest::Test
   attr_reader :parent, :invoice
   def setup
     sales_engine = SalesEngine.new
-    @parent = InvoiceRepository.new('test/support/sample_invoices.csv', sales_engine)
+    @parent = InvoiceRepository.new('test/support/invoices.csv', sales_engine)
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.